frontend/packages/dev-console/src/components/buildconfig/sections/ImagesSection.tsx

```diff
@@ -89,18 +89,18 @@ const ImagesSection: React.FC<{}> = () => {
           none: t('devconsole~None'),
           imageStreamTag: t('devconsole~Image Stream Tag'),
           imageStreamImage: t('devconsole~Image Stream Image'),
-          dockerImage: t('devconsole~Docker image'),
+          dockerImage: t('devconsole~External container image'),
```

Comment:
should image be Image[upper case I] as we have above `Image Stream Image`

![image](https://user-images.githubusercontent.com/5129024/189293789-1920c685-a1a0-42f5-99d6-01c146ba612f.png)

Reply:
My thought is that IST and ISI are resources, while External container image isn't. Wdyt?